stable/chartmuseum/values.yaml

```diff
@@ -34,6 +34,8 @@ env:
     STORAGE_GOOGLE_BUCKET:
     # prefix to store charts for google storage backend
     STORAGE_GOOGLE_PREFIX:
+    # gcp service-account
+    GOOGLE_SERVICE_ACCOUNT:
```

Comment:
Having GOOGLE_SERVICE_ACCOUNT in the `env.open` key will generate a corresponding environment variable : `GOOGLE_SERVICE_ACCOUNT=true`,  which is not used by chartmuseum. Should this be moved to `env.secret` maybe ? Everything that is present in the `env.open` is directly translated to env vars.

Reply:
Another note: GOOGLE_SERVICE_ACCOUNT is used in a boolean fashion, I think it would be clearer if it had a verb in the name: USE_GOOGLE_SERVICE_ACCOUNT maybe ?